directives/animate/animate.js

```diff
@@ -10,15 +12,13 @@ const moveYValue = 20;
 
 class AnimationState {
 
-	constructor(propertyPart) {
-
-		if (!(propertyPart instanceof PropertyPart) || propertyPart.committer.name !== 'animate') {
-			throw new Error('animation directives must be used with "animate" property');
+	constructor(partInfo) {
+		if (!(partInfo.type === PartType.PROPERTY) || partInfo.name !== 'animate') {
```

Comment:
In preparation for Lit 2, we swapped out PropertyPart for PartType, and take in a partInfo instead of a propertyPart. They have very similar fields, except partInfo does not have the committer field and you can access the properties directly.

Reply:
Once we're 100% on Lit 2, I think we can relax this check a bit. In Lit 2 you can just go `<div ${ hide() }>` and not tie it to a property at all, which is pretty sweet!